### DISCUSSION TITLE
v2.1

### DISCUSSION THREAD

**Discussion by Taiko2k** [Taiko2k/Avvie#30] [Announcements]
This update improves the layout of settings dialogue and adds a new circle output option.

### For packagers

Avvie now uses meson as a build system, so any existing build scripts will need to be rewritten.

<hr /><em>This discussion was created from the release <a href='https://github.com/Taiko2k/Avvie/releases/tag/v2.1'>v2.1</a>.</em>

**Comment by Solarunit** [comment]
I noticed a new flatpak dependency: `GNOME Software Development Kit`
Why is it needed? 
Because its size like ~500 MB iirc :open_mouth:

**Reply by Taiko2k** [reply]
Oh damn, that's a mistake that slipped by me. (A contributor submitted an updated manifest)

Ill push out a fix lightning fast.

**Reply by Solarunit** [reply]
Oh, thanks!  
Phew

**Comment by wilsonads** [comment]
Good afternoon, in the next version, you should add a magnifying glass option in the corners to make it more precise to crop an image. and that it moves with the cross or the plus sign.  and if you can put a totally black or dark dark mode that is #000000. Thanks for your time
